airbyte-config/config-models/src/main/java/io/airbyte/config/constants/AlwaysAllowedHosts.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.config.constants;
```

Comment:
Reviewers: Is this the right place to put something like this? 🤷 
This information will need to be used by the workers and possibly the orchestrator, so `airbyte-config` seemed like the right package.

Reply:
Looking at [Cole's chart](https://airbytehq-team.slack.com/archives/C04J3DJ0ADQ/p1674233851270909?thread_ts=1674233657.797829&cid=C04J3DJ0ADQ) of our current dependencies, seems like a reasonable spot if the orchestrator needs to get in on the action, otherwise `commons-worker` would probably be good?

Reply:
`2022` so last year...

This feels like something that would be a better fit in a configuration file than hard-coded.

Reply:
I have a suspicion that this list will become more complex than somethign we can statically represent... like... maybe based on which dataplane we are running on.